src/frontend/komponenter/Behandling/Inngangsvilkår/Medlemskap/InnflyttingUtflytting.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { IInnflyttingTilNorge, IUtflyttingFraNorge } from '../../../../typer/personopplysninger';
+import { Element, Normaltekst } from 'nav-frontend-typografi';
+import { Registergrunnlag } from '../../../Felleskomponenter/Visning/DataGrunnlagIkoner';
+import { StyledTabell } from '../../../Felleskomponenter/Visning/StyledTabell';
+import { formaterNullableIsoDato } from '../../../../utils/formatter';
+
+interface Props {
+    innflytting: IInnflyttingTilNorge[];
+    utflytting: IUtflyttingFraNorge[];
+}
+
+const InnflyttingUtflytting: React.FC<Props> = ({ innflytting, utflytting }) => {
+    return (
+        <StyledTabell kolonner={3}>
+            <Registergrunnlag />
+            <Element className="tittel" tag="h3">
+                Innflytting og utflytting
+            </Element>
+            {['Innflytting fra', 'Dato'].map((kolonneTittel, index) => (
+                <Element className={index === 0 ? 'førsteDataKolonne' : ''} key={index}>
```

Comment:
Skulle man kunna sätta className førsteDataKolonne  och så fiksa stylingen i css ? (. førsteDataKolonne:nth-of-type(1)).  Alternativt kan du bruke classNames({førsteDataKolonne: index === 0})